release-notes/major9/09.18.0/valtimo-backend-libraries.md

```diff
@@ -4,15 +4,31 @@
 
 The following features were added:
 
-* **New feature1**
+* **Added support for custom FormFieldDataresolvers.**
 
-  Description of the new feature goes here.
-  Also link to the page that explains the feature in greater detail.
+  Introducing support for creating ExternalFormFieldResolvers that implement a new support(String name) method.
+  The goal is to let implementation teams define their own FormFieldDataresolvers outside the ExternalFormFieldType enum definition.
+  Form keys should be able to define a key like so "anyDataSource.propertyName".
+  _Note: the old support(ExternalFormFieldType) method is being deprecated._
```

Comment:
There's a separate section on deprecations on this same page. Can you mention this deprecation there?